### ISSUE TITLE
Remove "Rate safe/questionable/explicit" options from mode menu

### ISSUE THREAD

**Issue by evazion** [r888888888/danbooru#3281]
Remove the `Rate safe` / `Rate questionable` / `Rate explicit` options from the mode menu. Sometimes users misrate posts because they stumble into the mode menu and randomly click on posts trying to get out.

I argue that mass rerating posts is almost never needed. If it is, you can use a tag script along with the `rating:` metatags. In fact, every action in the mode menu except approving and banning can now be achieved with tag scripts.

**Comment by BrokenEagle** [comment]
That's true... although some metatags aren't that well known, e.g. upvote, downvote locked, or that favorites can be added by adding fav: to the tag box.  Additionally, sometimes using the edit function is much faster than bothering with with tag script function.
